website/_dogfooding/docs-tests-sidebars.js

```diff
@@ -47,6 +48,17 @@ const sidebars = {
       collapsible: false,
       items: ['index', 'more-test'],
     },
+    {
+      type: 'category',
+      label: 'Category with only unlisted doc',
+      items: [
+        {
+          type: 'category',
+          label: 'Sub-category with only unlisted doc',
+          items: ['test-unlisted'],
```

Comment:
I'd rather create a new autogenerated category for unlisted with various test cases

Also maybe not reusing the same doc multiple times in the sidebar, it's not really a recommended practice even though it works fine